Core update page for 8.1 (#1590)
PrestaShop/docs: development/components/smarty-extensions/_index.md

### {url}

This helper is used to generate URLs.

> This will take care of http scheme (`http` or `https`), domain name, virtual and physical base URI, parameters concatenation, and of course URL rewriting.

Reviewer: ```suggestion
This will take care of the HTTP scheme (`http` or `https`), domain name, virtual and physical base URI, parameters concatenation, and URL rewriting.
```

| language          | ['id' => int]                   |                                                      |
| product           | ['id' => int]                   | ['category' => int, 'ean13' => string, 'ipa' => int] |
| category          | ['id' => int]                   | ['selected_filters' => array]                        |
| categoryImage     | ['id' => int]                   | ['selected_filters' => array, 'type' => int]         |
| manufacturer      | ['id' => int]                   |                                                      |
| manufacturerImage | ['id' => int]                   |                                                      |
| supplierImage     | ['id' => int]                   |                                                      |
| cms               | ['id' => int]                   |                                                      |
| module            | ['id' => int, 'name' => string] | ['controller' => string, 'params' => array]          |
| sf                | ['route' => string]             | ['sf-params' => array]                               |

and all other `ObjectModel` based entities (`address`, ...).

> `id` is almost always required to generate urls, except for the `sf` helper, where `route` is the required parameter. 

Reviewer: ```suggestion
`id` is almost always required to generate URLs, except for the `sf` helper, where `route` is the required parameter. 
```

> Other parameters have default values and can be overidden, eg.:

Reviewer: ```suggestion
Other parameters have default values and can be overridden, e.g.:
```
